profile/README.md

```diff
@@ -1,12 +1,98 @@
-## Welcome to CreoWis 👋
+<img src="creowis_banner_2.jpeg" alt="Banner" align="center"/>
 
-<!--
 
-**Here are some ideas to get you started:**
+<div align=center>
+  <h4> Questions? Ideas? Let’s chat! Feel free to reach out to us:</h4>
+
+  [![Website](https://img.shields.io/badge/Website-fff?logo=anaconda)](https://www.creowis.com)
+  [![G-Mail](https://img.shields.io/badge/Email-fff?logo=gmail)](mailto:hello@creowis.com)
```

Comment:
Can it be something else other than Gmail?

Reply:
M is the ideal symbol for mail and in shields.io only company logos are available so we will have to use this.